.gitignore

```diff
@@ -19,3 +19,4 @@ npm-debug.log
 
 .env/local
 dist/
+__snapshots__
```

Comment:
The jest docs says:
>The snapshot artifact should be committed alongside code changes
https://facebook.github.io/jest/docs/en/snapshot-testing.html#snapshot-testing-with-jest

is there any reason to ignore this folder?